src/lib.rs

```diff
@@ -1,5 +1,11 @@
-use core::panic;
-use std::cmp;
+pub mod apply_diff;
+pub use  crate::apply_diff::{apply_diff};
+pub mod hamming;
+pub use  crate::hamming::{HammingDistance};
+pub mod levenshtein;
+pub use crate::levenshtein::{LevenshteinDistance};
+
+
 #[doc = include_str!("../README.md")]
 #[derive(PartialEq, Eq, Debug)]
 pub enum StringDiffOpKind {
```

Comment:
Since the `doc` attribute is tied to the `StringDiffOpKind` enum, the Rustdoc tool will treat the README as the description for that enum.

 The doc attribute should be at the very top of the file instead (with an empty line after), and also needs a `!` for it to work (because it's considered an [inner attribute](https://doc.rust-lang.org/reference/attributes.html)), like:

```rs
#[doc = include_str!("../README.md")]

mod apply_diff;
// ...
```